build/Publish.targets

```diff
@@ -1,187 +1,79 @@
 
-      <FilesToPublish Include="@(SharedFxVersionBadge)">
-        <RelativeBlobPath>$(AliasBlobBasePath)%(SharedFxVersionBadge.FileName)%(SharedFxVersionBadge.Extension)</RelativeBlobPath>
+      <FilesToPublish Include="$(InstallersOutputPath)*.svg">
+        <RelativeBlobPath>$(BlobBasePath)%(FileName)%(Extension)</RelativeBlobPath>
         <CacheControl>no-cache, no-store, must-revalidate</CacheControl>
         <ContentType>image/svg+xml</ContentType>
         <Overwrite>true</Overwrite>
       </FilesToPublish>
 
-      <FilesToPublish Include="$(LatestRuntimeVersionFile)">
-        <RelativeBlobPath>$(AliasBlobBasePath)$(LatestRuntimeVersionFileName)</RelativeBlobPath>
-        <CacheControl>no-cache, no-store, must-revalidate</CacheControl>
-        <ContentType>text/plain</ContentType>
-        <Overwrite>true</Overwrite>
+      <!-- All other installer files. -->
+      <FilesToPublish Include="$(InstallerOutputPath)*" Exclude="@(FilesToPublish)">
+        <RelativeBlobPath>$(BlobBasePath)%(FileName)%(Extension)</RelativeBlobPath>
       </FilesToPublish>
 
-      <!-- Packages -->
-      <_PackageArtifactInfo
-        Include="@(ArtifactInfo)"
-        ArtifactPath="$(DependencyPackagesDir)%(ArtifactInfo.PackageId).%(ArtifactInfo.Version).nupkg"
-        Condition="'%(ArtifactInfo.ArtifactType)' == 'NuGetPackage'" />
-
-      <_SymbolsPackageArtifactInfo
-        Include="@(ArtifactInfo)"
-        ArtifactPath="$(DependencyPackagesDir)%(ArtifactInfo.PackageId).%(ArtifactInfo.Version).symbols.nupkg"
-        Condition="'%(ArtifactInfo.ArtifactType)' == 'NuGetSymbolsPackage'" />
-
-      <FilesToPublish Include="$(DependencyPackagesDir)%(ArtifactInfo.FileName)%(ArtifactInfo.Extension)" Condition="'%(ArtifactInfo.ArtifactType)' == 'JavaJar'">
-        <RelativeBlobPath>$(JarBlobBasePath)%(ArtifactInfo.FileName)%(ArtifactInfo.Extension)</RelativeBlobPath>
+      <!-- Java packages -->
+      <FilesToPublish Include="$(ProductPackageOutputPath)*.jar">
+        <RelativeBlobPath>$(JarBlobBasePath)%(FileName)%(Extension)</RelativeBlobPath>
       </FilesToPublish>
 
-      <FilesToPublish Include="$(DependencyAssetsDir)%(ArtifactInfo.FileName)%(ArtifactInfo.Extension)" Condition="'%(ArtifactInfo.ArtifactType)' == 'MavenPOM'">
-        <RelativeBlobPath>$(JarBlobBasePath)%(ArtifactInfo.FileName)%(ArtifactInfo.Extension)</RelativeBlobPath>
+      <FilesToPublish Include="$(ProductPackageOutputPath)*.pom">
+        <RelativeBlobPath>$(JarBlobBasePath)%(FileName)%(Extension)</RelativeBlobPath>
       </FilesToPublish>
 
-      <NpmPackageToPublish Include="$(DependencyAssetsDir)%(ArtifactInfo.FileName)%(ArtifactInfo.Extension)" Condition="'%(ArtifactInfo.ArtifactType)' == 'NpmPackage'">
-        <RelativeBlobPath>$(NpmBlobBasePath)%(ArtifactInfo.PackageId)/%(ArtifactInfo.FileName)%(ArtifactInfo.Extension)</RelativeBlobPath>
+      <!-- NPM packages -->
+      <NpmPackagesToPublish Include="$(ProductPackageOutputPath)*.tgz">
```

Comment:
Is it a safe assumption that all tgz files are npm packages?

Reply:
In the context of the ProductPackageOutputPath folder, yes, *.tgz should only match npm packages.